app/controllers/signatures/ongoing.js

```diff
@@ -16,12 +23,38 @@ export default class SignaturesOngoingController extends Controller {
       sort: {
         type: 'string',
       },
+      mandatees: {
+        type: 'array',
+      },
+      statuses: {
+        type: 'array',
+      }
+    },
+  ];
+
+  statusFilterItems = [
+    {
+      label: this.intl.t('signing-status-prepared'),
+      value: CONSTANTS.SIGNFLOW_STATUSES.PREPARED,
+    }, 
+    {
+      label: this.intl.t('signing-status-marked'),
+      value: CONSTANTS.SIGNFLOW_STATUSES.MARKED,
+    },
+    {
+      label: this.intl.t('signing-status-signed'),
+      value: CONSTANTS.SIGNFLOW_STATUSES.SIGNED,
```

Comment:
The idea of adding the `SIGNFLOW_STATUSES` constants in `app/config/constants.js` was so you could loop over them, while we could still control which statuses to show in the filter and which to hide (at the time of the ticket we only supported 3 statuses). Even that is obsolete in the meantime as we now support all statuses, so you can just get them from the database.

These are concept resources with a label in the database, so there's no need for translation keys.

I changed this so `statusFilterItems` is collected from the resources store using the conceptScheme for the statuses. Also split off the status filter in a component for easier reuse and cleaner code